cg/cli/downsample.py

```diff
@@ -0,0 +1,81 @@
+"""cg module for downsampling reads in a sample."""
+
+import logging
+from typing import List
+
+import click
+
+from cg.apps.downsample.downsample import DownSampleAPI
+from cg.apps.downsample.utils import format_downsample_case
+from cg.constants.constants import DRY_RUN
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def downsample_cmd():
+    """Downsample reads in a sample."""
+
+
+@downsample_cmd.command("samples", help="Downsample reads in one or multiple samples in a case.")
+@click.option(
+    "-c",
+    "--case-internal-id",
+    required=True,
+    help="Case identifier used in statusdb, e.g. supersonicturtle",
+)
+@click.option(
+    "-sr",
+    "--sample_internal_id_reads",
+    required=True,
+    multiple=True,
+    help="Identifier used in statusdb, e.g. ACC1234567 and the number of reads to down sample to in millions separated by ;."
+    "e.g. ACC1234567;30",
+)
+@DRY_RUN
+def downsample_sample(
+    context: CGConfig, case_internal_id: str, sample_internal_id_reads: List[str], dry_run: bool
```

Comment:
why is there a case_id in the downsample_sample command?

Reply:
A sample cannot exist without a case for an analysis for which the down sampled sample will be used. Outlined in the issue